contrib/win32/win32compat/fileio.c

```diff
@@ -673,6 +673,25 @@ WriteCompletionRoutine(_In_ DWORD dwErrorCode,
 	*((__int64*)&lpOverlapped->Offset) += dwNumberOfBytesTransfered;
 }
 
+
+int 
+fileio_write_wrapper(struct w32_io* pio, const void* buf, size_t max_bytes) 
+{
+	if (max_bytes <= WRITE_BUFFER_SIZE)
```

Comment:
shouldn't this be 
if (max_bytes < WRITE_BUFFER_SIZE)?

Reply:
fileio_write works if the buffer size is WRITE_BUFFER_SIZE though.